server/routes/alpha/activity-list/activityListRoutes.ts

```diff
@@ -0,0 +1,18 @@
+import { RequestHandler, Router } from 'express'
+import asyncMiddleware from '../../../middleware/asyncMiddleware'
+import ActivityListRouteHandler from './handlers/activityListRouteHandler'
+import SelectActivityLocationRouteHandler from './handlers/selectActivityLocationRouteHandler'
+import { Services } from '../../../services'
+
+export default ({ prisonService }: Services): Router => {
+  const router = Router()
+
+  const selectActivityLocation = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  const selectActivityLocationRouteHandler = new SelectActivityLocationRouteHandler(prisonService)
+  selectActivityLocation('/select-activity-location', selectActivityLocationRouteHandler.GET)
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
```

Comment:
This only needs defined once, you dont need to define `selectActivityLocation` too - they are identical

Reply:
sorry i dont understand this one - can you help

Reply:
Replied with suggested code change

Reply:
Got it - thanks